units/commit_queue.go

```diff
@@ -66,6 +66,32 @@ func NewCommitQueueJob(env evergreen.Environment, queueID string, id string) amb
 	return job
 }
 
+func (j *commitQueueJob) TryUnstick(cq *commitqueue.CommitQueue) {
+	//unstuck the queue if the patch is done.
+	nextItem := cq.Next()
+	if nextItem != nil {
+		patch, err := patch.FindOne(patch.ById(patch.NewId(nextItem.Issue)).WithFields(patch.FinishTimeKey))
```

Comment:
Can she get the version instead of the patch?

Also (sorry to overload), typically we try to avoid variable names that are the same as packages (in this case, "patch").

Reply:
we could try. There's a short window when processing is true and a version hasn't been created yet.